crates/core/tests/integration_datafusion.rs

```diff
@@ -1250,13 +1250,14 @@ mod local {
     #[tokio::test]
     async fn test_issue_2105() -> Result<()> {
         use datafusion::arrow::datatypes::{DataType as ArrowDataType, Field, Schema};
-        let path = tempfile::tempdir().unwrap();
-        let path = path.into_path();
+        let tmp_dir = tempfile::tempdir().unwrap();
+        let path = tmp_dir.path();
```

Comment:
this is a small drive by - it seems that `into_path` would persist things to disk and no longer clean up the storage once done.